isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -59,64 +161,57 @@ one time) may be useful depending on the complexity of your processing requireme
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#new-environmental-variable-meanings">New ISIS3 environmental variables and their meanings</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#custom-data-and-test-data-directories">Custom data and test directories</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#cleaning-builds">Cleaning builds</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#building-individual-isis3-applicationsobjects">Building individual applications/objects</a></li>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#building-isis3-documentation">Building ISIS 3 documentation</a></li>
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#building-isis3-documentation">Building ISIS3 documentation</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#problems">What to do if you encounter any problems</a></li>
   </ul>
 </p>
 
-<h3>Links for Developers</h3>
 
-  <ul>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Developer-Checklists">ISIS 3 Developer Checklist</a></li>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Jenkins-CI">Jenkins Continuous Integration Testing</a></li>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Managing-Custom-ISIS-Dependencies-(WIP)">Managing Custom ISIS 3 Dependencies</a></li>
-  </ul>
-
-<h3>Full ISIS 3 Data Download</h3>
+<h3>Full ISIS3 Data Download</h3>
 
 <p>
-    If you have the disk space and network speed, you may want to download all of the ISIS data areas which
-    includes all missions supported by ISIS.   This takes over 130 GB of disk space!  If you only have a 10 Mbps
-    network connection it will take nearly two days to download.  If you want to acquire only certain mission data
-     <a href="#MissionSpecific">click here</a>.   To download all ISIS 3 data files continue reading.
+    Mission data is hosted on rsync servers and not through conda channels like the ISIS3 distribution.
+    This requires using the rsync command from within a terminal window within your Unix distribution, or
+    from within WSL if running Windows 10.   Downloading all mission data requires over 130 GB of disk space.
+    If you want to acquire only certain mission data <a href="#MissionSpecific">click here</a>.
+    To download all ISIS3 data files, continue reading.
 </p>
 <p>
-    Remember to use the following command from the same directory you ran the previous rsync
-    command. In the example it was "/work1/isis3".
+    To download all ISIS3 data, enter the following commands at the command prompt:
```

Comment:
Should there be a warning about how large the entire data set is?

Reply:
There are people within Astro (like Kaj) who find these instructions useful.  Here I am emulating what the SPICE team does in their public disclaimer concerning wrappers like spiceypy - i.e. we're not supporting this, but it has been found useful for other users.  As long as we're clear that we don't support Windows 10 and will not be resolving bugs with this, then I think it's ok.

Reply:
Yes, we probably should add a line about how sizable the download is.